src/main/java/io/debezium/connector/jdbc/JdbcSinkConnectorConfig.java

```diff
@@ -278,6 +280,24 @@ public class JdbcSinkConnectorConfig {
             .withDescription("Specifies how many records to attempt to batch together into the destination table, when possible. " +
                     "You can also configure the connector’s underlying consumer’s max.poll.records using consumer.override.max.poll.records in the connector configuration.");
 
+    public static final Field FLUSH_MAX_RETRIES_FIELD = Field.create(FLUSH_MAX_RETRIES)
+            .withDisplayName("Max retry count")
+            .withType(Type.INT)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTOR_ADVANCED, 5))
+            .withWidth(ConfigDef.Width.SHORT)
+            .withImportance(ConfigDef.Importance.LOW)
+            .withDefault(5)
+            .withDescription("Max retry count when fail to flush. retry if retriable exceptions");
```

Comment:
```suggestion
            .withDescription("Max retry count when fail to flush. In case of a retryable exceptions, the flush will be retried for the defined max retry count. Default is 5.");
```